### PR TITLE
OCPBUGS-19541: Console Operand not working for clusterwide CRDs

### PR DESCRIPTION
**Pull request by jpinsonneau**
This PR allows using descriptors for clusterwide CRDs.
Check https://issues.redhat.com/browse/OCPBUGS-19541 for details

Before:
image
After:
image

### CODE REVIEW

**Comment by TheRealJon** [comment]
I'm a little bit hesitant about this because it might be a fragile fix. 

TLDR; We previously considered and decided against this approach because of the technical limitations of the API as well as UX and RBAC concerns, but it's possible we might reconsider given there is a need.

In this case, We have a cluster-scoped operand that is managed by a namespace-scoped CSV. This PR adds the name of the CSV to the route for cluster-scoped operands. There are a few implications we need to consider:
- This breaks our convention of '/k8s/cluster' routes always referring to cluster-scoped resources, which is not ideal. 
- We have no namespace parameter to provide to the request for CSVs. Due to technical limitations, this results in a cluster-scoped list request for all CSVs, with a `metadata.name=<name from URL>` field selector. It's technically possible for multiple CSVs to exist with the same name, so we can't guarantee a determinate result.
- There is at least one RBAC implication. In order for this solution to work, a user needs cluster-scoped list permissions on CSVs, which is a fairly significant escalation for users who don't have this already.
- Probably most importantly, descriptors are legacy. I'm not sure adding changes to a legacy API is the right thing. We are encouraging operator authors to migrate to dynamic plugins as much as possible because this provides a much higher degree of control over content.

All of this being said, most of these concerns might be small enough edge cases that we could still consider using the solution.

@jhadvig @spadgett Do you have any comments?
